src/Okta.Sdk/Generated/ApplicationsClient.Generated.cs

```diff
@@ -259,7 +259,7 @@ public async Task<IAppUser> AssignUserToApplicationAsync(IAppUser appUser, strin
                 }, cancellationToken).ConfigureAwait(false);
         
         /// <inheritdoc />
-        public async Task DeleteApplicationUserAsync(string appId, string userId, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task DeleteApplicationUserAsync(string appId, string userId, bool? sendEmail = false, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Is this signature change a breaking change?

Reply:
It could be. As it has a default value, if people are using this method by passing only `appId` and `userId` they will be fine, otherwise, their code will not compile.
What should we do in these cases?

Reply:
Sounds like a breaking change to me.  Either we major rev to 2.0, or we find a way to add this without breaking.  Is it possible to add the new parameters at the end of the signature, without causing compile errors?